src/platform/forms-system/src/js/containers/FormPage.jsx

```diff
@@ -166,10 +168,11 @@ class FormPage extends React.Component {
   }
 }
 
-function mapStateToProps(state) {
+function mapStateToProps(state, ownProps) {
   return {
     form: state.form,
     user: state.user,
+    ancillaryData: ownProps.route.pageConfig.ancillaryData(state),
```

Comment:
Do we need to check if this selector exists first?

Reply:
Ahh, good point. Yes. It didn't break because this `mapStateToProps` isn't used; the unconnected component is used in `RoutedSavablePage`. :+1: